packages/core/src/components/templates/ProductListingPage/ProductListingPage.tsx

```diff
@@ -70,69 +81,95 @@ const useSearchParams = ({
   return useMemo(() => parseSearchState(new URL(hrefState)), [hrefState])
 }
 
+const overwriteMerge = (_, sourceArray, __) => sourceArray
```

Comment:
```suggestion
const overwriteMerge = (_, sourceArray) => sourceArray
```

Also add that other comment here.

Reply:
It was added along with the one from pdp [here](https://github.com/vtex/faststore/pull/1967/commits/16eedfc7b108394fe99a7e5c2bc63fa72ee365de).